### PR TITLE
Remove `AdminServiceEventStore`-related features

### PR DESCRIPTION
**Pull request by shannynalayna**
Removes instances of the `admin-service-event-store` and `admin-service-event-store-postgres`, as well as code guarded with `not(feature = "admin-service-event-store")`.  Also makes a small fix that came up when running `just lint` after the feature was removed to add a feature guard for an import statement in the `add_event` module.

### CODE REVIEW

File: libsplinter/Cargo.toml [hunk 1]
```diff
@@ -105,7 +105,6 @@ experimental = [
     # The following features are experimental:
     "admin-service-client",
     "admin-service-event-store",
```
Comment (vaporos) [inline libsplinter/Cargo.toml:107]: Did you intend to also remove admin-service-event-store?

Comment (shannynalayna) [reply libsplinter/Cargo.toml]: Yes, I tried to separate removing the two features and forgot this line when I updated the PR - fixed now

File: libsplinter/src/admin/rest_api/actix/ws_register_type.rs [hunk 2]
```diff
@@ -317,24 +231,10 @@ struct JsonAdminEvent {
...
-            event: raw_evt.1,
-            event_id: None,
-        }
-    }
-}
-
 // Conversion for the `JsonAdminEvent` sent to subscribers when the `AdminServiceStore` is
 // used to store admin events.
-// `timestamp` is set to the `UNIX_EPOCH` value to allow for backward-compatibility, as the
+// `timestamp` is set to the current time to allow for backward-compatibility, as the
```
Comment (vaporos) [inline libsplinter/src/admin/rest_api/actix/ws_register_type.rs:236]: Is this change intentional?

Comment (shannynalayna) [reply libsplinter/src/admin/rest_api/actix/ws_register_type.rs:236]: Yes, because that comment was not valid before as the timestamp was being set to the current time (using `SystemTime::now()`) rather than the `UNIX_EPOCH` time value.

File: libsplinter/src/admin/store/diesel/operations/add_event.rs [hunk 3]
```diff
@@ -21,13 +21,15 @@ use diesel::{dsl::insert_into, prelude::*};
 use super::AdminServiceStoreOperations;
 
 use crate::admin::service::messages;
+#[cfg(feature = "sqlite")]
```
Comment (vaporos) [inline libsplinter/src/admin/store/diesel/operations/add_event.rs:24]: Should the two implementations here actually differ?

Comment (shannynalayna) [reply libsplinter/src/admin/store/diesel/operations/add_event.rs:24]: They do, in how the event is initially added, then the event_id is retrieved. If you look at how the postgres impl is getting the `event_id`: https://github.com/Cargill/splinter/blob/main/libsplinter/src/admin/store/diesel/operations/add_event.rs#L63, versus the sqlite impl: https://github.com/Cargill/splinter/blob/main/libsplinter/src/admin/store/diesel/operations/add_event.rs#L155

Comment (shannynalayna) [reply libsplinter/src/admin/store/diesel/operations/add_event.rs:24]: Also, generally the `add_*` methods can't use the generic connection, if you look at the other add operations implemented for the admin store. (Most have separate implementations for the postgres and sqlite feature, though the implementations don't differ greatly)

**Review by peterschwarz — APPROVED** [review]

**Review by vaporos — APPROVED** [review]

**Review by isabeltomb — APPROVED** [review]

**Review by agunde406 — APPROVED** [review]

**Review by ltseeley — APPROVED** [review]

**Review by RyanLassigBanks — APPROVED** [review]

**Review by dplumb94 — APPROVED** [review]

**Review by dnewh — APPROVED** [review]

**Review by rbuysse — APPROVED** [review]
